src/main/resources/GoogleApps/Groovy.groovy

```diff
@@ -361,129 +363,133 @@ public class GoogleAppsGroovy {
                 addDebug("Wrong domain: Removed credentials for userid " + userId)
                 return -1;
             }
-        } else {
-            String id = user.get("resourceName");
-            if(id.startsWith("people/") && id.length()>7) id = id.substring(7);
-            String email = "";
-            def db = context.getDatabase()
-            try {
-                // Force main wiki database to create the user as global
-                context.setDatabase("xwiki")
+        }
+        String id = user.get("resourceName");
+        if(id.startsWith("people/") && id.length()>7) id = id.substring(7);
```

Comment:
Our best practice is to always wrap the IF body in a block ``{...}``.

Reply:
I've applied this convention for this file. Note that this is not the only violation and we should care for such style best practice in the java part.